ci/jenkins/jobs/macros.yaml

```diff
@@ -328,10 +328,10 @@
           export KUBECONFIG=$WORKSPACE/jenkins/out/kubeconfig
 
           kubectl apply -f build/yamls/antrea.yml
-          kubectl rollout restart deployment/coredns -n kube-system
-          kubectl rollout status deployment/coredns -n kube-system
-          kubectl rollout status deployment.apps/antrea-controller -n kube-system
-          kubectl rollout status daemonset/antrea-agent -n kube-system
+          timeout 5m kubectl rollout restart deployment/coredns -n kube-system
```

Comment:
can `kubectl rollout restart` get stuck?

Reply:
Thanks Antonin, I tried the "kubectl rollout rest" and it won't be stuck. Remove timeout from it.